src/api/app/components/request_decision_component.html.haml

```diff
@@ -1,4 +1,9 @@
 .request-decision
+  - if policy(@bs_request).handle_request? && @show_project_maintainer_hint
```

Comment:
:thought_balloon: You could move this condition into the class

Reply:
Done, I moved directly into the component initialization